cluster/images/etcd/migrate/options.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 var (
-	supportedEtcdVersions = []string{"3.0.17", "3.1.20", "3.2.32", "3.3.17", "3.4.18", "3.5.7"}
+	supportedEtcdVersions = []string{"3.0.17", "3.1.20", "3.2.32", "3.3.17", "3.4.18", "3.5.8"}
```

Comment:
I have no idea. I just know current support "3.4.x" and "3.5.x".

Reply:
> are all these versions of etcd compatible ?

etcd versions 3.3.x and below are out of support.  From client perspective, both 3.4 and 3.5 should be compatible with the etcd `clientv3` API.

Reply:
> > are all these versions of etcd compatible ?
> 
> etcd versions 3.3.x and below are out of support. From client perspective, both 3.4 and 3.5 should be compatible with the etcd `clientv3` API.

Thanks.

Reply:
If i understand correctly, `supportedEtcdVersions` should be change to `"3.0.17", "3.1.20", "3.2.32", "3.3.17", "3.4.18", "3.5.8"`.
Those versions which contain `"3.0.17", "3.1.20", "3.2.32", "3.3.17", "3.4.18"` are used for lower version kubernetes. 
So we should change https://github.com/kubernetes/kubernetes/blob/master/cluster/images/etcd/migrate/options.go#L31
to  `"3.0.17", "3.1.20", "3.2.32", "3.3.17", "3.4.18", "3.5.8"`. 

@pacoxu @dims  Could you please take a look？

Reply:
`supportedEtcdVersions` provides a default value for the `--bundled-versions` flag.  Limiting it to the latest two minor versions seems appropriate, since you need at least two minor versions if you're doing a migration, but might not have more than that.

I can also see an argument for making this flag required, but that's a bigger change, and this should all go away soon.

Maybe just leave a comment above `supportedEtcdVersions` stating that, for now, we're setting it to the latest patch of the most recent two minor versions?

Reply:
OK.  When i compare here and  https://github.com/kubernetes/kubernetes/blob/a5575425b039bf7c15dfaa9a7acf257fdc4fde3f/cluster/images/etcd/Makefile#L29, 
I have some doubts， we should fix in https://github.com/kubernetes/kubernetes/issues/117906